### PR TITLE
backport: Merge bitcoin#22619, 22593

### PR DESCRIPTION
**Pull request by vijaydasmp**
bitcoin backports

### CODE REVIEW

**Comment by vijaydasmp** [comment]
Hello @UdjinM6 , @knst , @PastaPastaPasta for review

**Review by UdjinM6 — CHANGES_REQUESTED** [review]
pls see https://github.com/UdjinM6/dash/commit/35b39cfee30395d1613eeb7b672027ed31901637

File: doc/release-notes-22570.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Files
+-----
+
+* On startup, the list of banned hosts and networks (via `setban` RPC) in
+  `banlist.dat` is ignored and only `banlist.json` is considered. Dash Core
+  version 22.x is the only version that can read `banlist.dat` and also write
```
Comment (knst) [inline doc/release-notes-22570.md:6]: 22570 has been already released with version Dash Core v20: https://github.com/dashpay/dash/pull/5574


DNM 22603

**Review by kwvg — CHANGES_REQUESTED** [review]
Please backport bitcoin#22593 as well (see https://github.com/dashpay/dash/commit/cb1745bb62f97513f580a2980bf18e6de022051b).

**Review by UdjinM6 — APPROVED** [review]
utACK 26ec2e4a14045291c96c7e2dc04db9c318599977

**Comment by vijaydasmp** [comment]
> utACK [26ec2e4](https://github.com/dashpay/dash/commit/26ec2e4a14045291c96c7e2dc04db9c318599977)

had to rebase , requesting re-review

**Review by UdjinM6 — APPROVED** [review]
utACK a6ec9bcc3793e0579ee039a204c6d971cfbc797e

**Review by knst — APPROVED** [review]
utACK ad840ec31aca0156272dde9f1bda523bbd444edb

**Review by kwvg — APPROVED** [review]
utACK ad840ec31aca0156272dde9f1bda523bbd444edb

**Pull request merged**